styles/components/UploadList.less

```diff
@@ -34,14 +34,16 @@
   padding-top: 20px;
   // room for scrollbar so it doesn't overlap content
   padding-right: 15px;
-  max-height: 260px;
+  min-height: 370px;
   overflow-y: scroll;
 }
 
 .UploadList-item {
-  margin-bottom: 30px;
+  padding: 15px 0;
+  border-bottom: 1px solid @gray-light;
```

Comment:
I like the hairline.

I wonder if we can tighten up the vertical space between the type-ins and
buttons for each device group a bit, e.g. put username/password/import
closer together for CareLink, etc for choose file/upload for OmniPod.


On Thu, Feb 19, 2015 at 7:40 AM, Jana Beck <notifications@github.com> wrote:

> In styles/components/UploadList.less
> <https://github.com/tidepool-org/chrome-uploader/pull/75#discussion_r24968565>
> :
>
> >    overflow-y: scroll;
> >  }
> >
> >  .UploadList-item {
> > -  margin-bottom: 30px;
> > +  padding: 15px 0;
> > +  border-bottom: 1px solid @gray-light;
>
> How do we feel about using a hairline separator for both separating the
> groups dropdown and between each upload list item? CC @HowardLook
> <https://github.com/HowardLook> @brandonarbiter
> <https://github.com/brandonarbiter>
>
> The separator is a little wider below the groups dropdown, but otherwise
> it's very similar (IMO).
>
> [image: screenshot 2015-02-18 22 37 54]
> <https://cloud.githubusercontent.com/assets/1588547/6262780/1519a326-b7bf-11e4-944c-de7136455bab.png>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tidepool-org/chrome-uploader/pull/75/files#r24968565>.
>